src/mechanisms/scram/parser.rs

```diff
@@ -26,7 +26,7 @@ impl MechanismError for SaslNameError {
     }
 }
 
-#[derive(Copy, Clone)]
```

Comment:
Thanks!

Do feel free add a 'closes #20' to the description of your commit(s) so that merging this PR will automatically close that issue :)